DDG4/src/Geant4StepHandler.cpp

```diff
@@ -125,6 +125,7 @@ double Geant4StepHandler::birkAttenuation() const    {
   const G4Track* trk = step->GetTrack();
   const G4ParticleDefinition* particle = trk->GetDefinition();
   const G4MaterialCutsCouple* couple = trk->GetMaterialCutsCouple();
+  s_emSaturation.SetVerbose(true);
```

Comment:
a) SetVerbose takes an int, http://www-geant4.kek.jp/Reference/10.02.p02/classG4EmSaturation.html
b) It would be better to set the argument in line 117 to `1`, and only call `SetVerbose(1)` for older geant versions. by putting the call in the if/else pre-processor block.
Then we can avoid calling this function for every step at least in newer geant4 versions.